stylesheets/all.scss

```diff
@@ -1,79 +1,99 @@
-@import 'node_modules/govuk-frontend/govuk/all';
+@import 'node_modules/lbh-frontend/lbh/all';
 @import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600&amp;display=swap');
 
 $screen-sm-min: 576px;
 
 * {
-  font-family: Open Sans, Arial, sans-serif !important;
+  font-family: 'Open Sans', sans-serif;
 }
 
-[role='button'] {
-  cursor: pointer;
+.lbh-phase-banner {
+  margin-top: 5px;
 }
 
-.govuk-tag:not([class*='govuk-tag--']) {
-  background-color: #00513f;
+.clear-button {
+  display: inline-block;
+  border: none;
+  background: none;
+  @include lbh-body-m;
+  color: lbh-colour('lbh-link');
+  cursor: pointer;
+  margin: 2.3rem 1.375rem 0.625rem;
 }
 
-.govuk-required {
-  color: #d21b1b;
+.section-heading {
+  padding-bottom: 12px;
+  border-bottom: 3px solid lbh-colour('lbh-text');
+  margin-bottom: 10px;
 }
 
-.govuk-table__row--clickable:hover {
-  background: govuk-colour('light-grey');
-  cursor: pointer;
+.lbh-back-link:visited {
+  color: lbh-colour('lbh-link');
 }
 
-.govuk-input:disabled {
-  background: govuk-colour('light-grey');
+.lbh-button--add {
+  svg {
+    margin-right: 10px;
+  }
+  svg rect {
+    fill: lbh-colour('lbh-primary-button');
+  }
+  &:hover svg rect {
+    fill: lbh-colour('lbh-secondary-button');
+  }
 }
 
-.govuk--timestamp {
-  font-weight: 600;
-}
+// new stuff above here
 
-.gov-weight-lighter {
-  font-weight: 100;
-}
+// [role='button'] {
+//   cursor: pointer;
+// }
 
-.govuk-custom-text-color {
-  color: #01674e;
-}
+// .govuk-tag:not([class*='govuk-tag--']) {
+//   background-color: #00513f;
+// }
 
-.govuk-divider {
-  border: solid 2px #01674e;
+.govuk-required {
+  color: lbh-colour('lbh-error');
 }
 
-.govuk-warning {
-  color: #da3711;
-}
+// .govuk-table__row--clickable:hover {
+//   background: govuk-colour('light-grey');
+//   cursor: pointer;
+// }
```

Comment:
Can we get rid of the commented out code?

Reply:
not sure if we might need some of it yet - i can't see it doing anything but there are pages i'm not familiar with. we could always bring it back from older commits i support?

Reply:
removed (for now)